circle.yml

```diff
@@ -12,6 +12,6 @@ test:
     - make test
     - nvm install 0.12.7
     - nvm alias default 0.12.7
```

Comment:
You could do `nvm use 0.12.7 && make test` once the install is done though. There's no need for that `default alias` bit.